### DISCUSSION TITLE
Filter in Relationshipfield

### DISCUSSION THREAD

**Discussion by Kakovisk** [Budibase/budibase#9729] [Help] [answered]
Hello everyone,

I have the following situation:

In a Side Panel for New Accounts included, the user needs to choose a Bank in the relationship field.
![img 1](https://user-images.githubusercontent.com/112348859/219794226-21a235d3-bbb2-4c31-8c59-b2b5943c8f2a.png)
![img 2](https://user-images.githubusercontent.com/112348859/219794237-4aa17525-fb25-4a29-a0c1-30f83ec42cfb.png)

But I would like you to display the banks with the condition fk_empresas_bancos = 2.

![img 3](https://user-images.githubusercontent.com/112348859/219794295-5e7a8305-fc4b-48d2-8540-3d47094f799d.png)

Eu terei que fazer um controle a parte para essa situação ?

Tks.

**Comment by melohagan** [comment, marked as answer]
Hey @Kakovisk 

I think what you are looking to do is filter a relationship picker, which is currently not possible but there's an open issue here: https://github.com/Budibase/budibase/issues/7810
